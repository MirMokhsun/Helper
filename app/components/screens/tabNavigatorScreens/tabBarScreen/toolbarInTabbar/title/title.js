import React, { PureComponent } from 'react';
import { Text, View } from 'react-native';
import { styles } from './style';

export class Title extends PureComponent {
  constructor(props) {
    super(props)
  }

  render() {
    const { pageName } = this.props;
    return (
      <View style={styles.container}>
        <Text style={styles.text} testID="TabTitleID" accessibilityLabel="TabTitleID">{pageName}</Text>
      </View>
    );
  }
}
