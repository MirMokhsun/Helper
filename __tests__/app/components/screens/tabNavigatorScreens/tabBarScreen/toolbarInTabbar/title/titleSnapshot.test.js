import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { Title } from '../../../../../../../../app/components/screens/tabNavigatorScreens/tabBarScreen/toolbarInTabbar/title/title';

it('create snapshot ToolbarInTabbar', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<Title
        pageName={'Help'}
    />);
    expect(toolbarSnapshot).toMatchSnapshot();
});