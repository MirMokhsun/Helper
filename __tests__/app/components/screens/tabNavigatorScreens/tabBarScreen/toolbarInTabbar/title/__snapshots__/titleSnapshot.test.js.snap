// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot ToolbarInTabbar 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessibilityLabel="TabTitleID"
    style={
      Object {
        "color": "#000756",
        "fontFamily": "Roboto-Light",
        "fontSize": 20,
      }
    }
    testID="TabTitleID"
  >
    Help
  </Text>
</View>
`;
